src/routes/+layout.svelte

```diff
@@ -17,9 +17,9 @@
 </script>
 
 <NavBar>
-  <div
-    class="h-[calc(100vh-9rem)] flex-col overflow-auto pb-8 accent-primary md:h-[calc(100vh-5rem)] [&>*]:flex-1"
-  >
+  <div class="overflow-auto bg-base-200 accent-primary" style="height: 4.4rem"></div>
```

Comment:
We try and stick exclusively to Tailwind classes. In this case, something like `h-16` would be equivalent to the style attribute used.